### DISCUSSION TITLE
error NET::ERR_CERT_AUTHORITY_INVALID codeload.github.com

### DISCUSSION THREAD

**Discussion by ManuelRCastroIglesias** [community/community#45514] [Projects and Issues]
When I try to download a repository in ZIP, (~~https~~://codeload.github.com/...) I get a certificate error.  
![Captura de pantalla 2023-01-27 105101 (1)](https://user-images.githubusercontent.com/100691894/215061908-15eadcc2-79e3-43b5-a8bd-a44b69529a24.jpg)

How can I fix it?

For a few days I have been receiving the following error:

----
Su conexión no es privada.
Es posible que los atacantes estén intentando robar tu información de codeload.github.com (por ejemplo contraseñas, mensajes o tarjetas de crédito).
NET::ERR_CERT_AUTHORITY_INVALID
codeload.github.com usa cifrado para proteger la información. Cuando Microsoft Edge intentó conectarse a codeload.github.com esta vez, el sitio web envió credenciales poco comunes e incorrectas. Esto puede ocurrir cuando un atacante está intentando hacerse pasar por codeload.github.com o una pantalla de inicio de sesión de Wi-Fi ha interrumpido la conexión. La información sigue protegida porque Microsoft Edge detuvo la conexión antes de que se intercambiaran datos.

No puedes visitar codeload.github.com en este momento porque el sitio web usa HSTS. Los ataques y los errores de red suelen ser temporales, por lo que esta página probablemente funcionará más tarde.

----
Your connection is not private.
Attackers may be trying to steal your codeload.github.com information (such as passwords, messages, or credit cards).
NET::ERR_CERT_AUTHORITY_INVALID

codeload.github.com used encryption to protect information. When Microsoft Edge tried to connect to codeload.github.com this time, the website sent unusual and incorrect credentials. This can occur when an attacker is trying to impersonate codeload.github.com or a Wi-Fi login screen has interrupted the connection. The information remains protected because Microsoft Edge stopped the connection before data was exchanged.

You can't visit codeload.github.com right now because the website uses HSTS. Network attacks and errors are usually temporary, so this page will probably work later.

**Comment by jge162** [comment]
Hello @ManuelRCastroIglesias thanks for your question. 
a. Try clearing your browser's cache and cookies
b. Try accessing the website using a different browser
c. Check your internet connection and try again later
Id. f the problem persist, try to download the repository via Git.

you can also submit a support ticket for assistance. [bug report](https://support.github.com/contact/bug-report)
and support will get back in touch with you. be patient

**Reply by ManuelRCastroIglesias** [reply]
@jge162 Ni Edge, ni Google Chrome.  
Reinicio limpio tampoco, ¡Creía que aquí era el sitio adecuado para pedir ayuda!  
¡GRACIAS!

**Reply by jge162** [reply]
no problem.

**Comment by ManuelRCastroIglesias** [comment]
@jge162 
I finally found what the problem was! 

It was all due to my internet provider (Vodafone) which has a security system to prevent access to possibly malicious sites (sometimes they are not).

That system blocked access to codelod.github.com

After several calls to that company and a couple of days fighting the system, we managed to get them to desctivate some of that protection (at least for me).

With what I already have again working, reasonably well, HitHub.

I hope that if someone finds themselves in the same situation,  
my hope will inspire them to find the solution.

**Reply by Roccoz2004** [reply]
I really hope I don't have to call Spectrum over this ridiculousness. I am having the same problem.

**Reply by Thiamath** [reply]
This was the answer for me too, I changed to the mobile hotspot, and worked like a charm.
Thinking about burning Vodafone to ashes after 4 hours banging my head against this.
